thirdparty/prod/schematics.ABOUT

```diff
@@ -0,0 +1,11 @@
+about_resource: schematics-1.1.1-cp27-none-any.whl
+version: 1.1.1
+
+name: schematics
+home_url: http://github.com/schematics/schematics
+owner: James Dennis
+dje_license: bsd-new
+notice_file: schematics.NOTICE
```

Comment:
We need also a `license_text_file: schematics.LICENSE` and the  file and a `copyright:`